inputFiles/hydraulicFracturing/scripts/hydrofractureFigure.py

```diff
@@ -0,0 +1,220 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import xml.etree.ElementTree as ElementTree
+import HydrofractureSolutions
+import sys
+
+
+def getParametersFromXML(xmlFilePath):
+    prefix = "../../../../../../../inputFiles/hydraulicFracturing/"
+
+    tree = ElementTree.parse(prefix + xmlFilePath + "_benchmark.xml")
+
+    maxTime = float(tree.find('Events').get('maxTime'))
+
+    param = tree.findall('Geometry/Box')
+    found_source = False
+    for elem in param:
+        if elem.get("name") == "source":
+            source = elem.get("xMax")
+            source = [float(i) for i in source[1:-1].split(",")]
+            x_source = round(source[0])
+            found_source = True
+        if found_source: break
+
+    tree = ElementTree.parse(prefix + xmlFilePath + "_base.xml")
+
+    elasticParam = tree.find('Constitutive/ElasticIsotropic')    
+
+    viscosity = float(tree.find('Constitutive/CompressibleSinglePhaseFluid').get('defaultViscosity'))    
+
+    fluidDensity = float(tree.find('Constitutive/CompressibleSinglePhaseFluid').get('defaultDensity'))
+
+    if xmlFilePath == 'kgdToughnessDominated' or xmlFilePath == 'kgdViscosityDominated':
+
+       youngModulus = float(elasticParam.get('defaultYoungModulus'))
+       poissonRatio = float(elasticParam.get('defaultPoissonRatio'))
+       injectionRate = -2.0 * float(tree.find('FieldSpecifications/SourceFlux').get('scale')) / fluidDensity
+       tree = ElementTree.parse(prefix + xmlFilePath + "_base.xml")
+       toughness = float(tree.find('Solvers/SurfaceGenerator').get('rockToughness'))
+
+    elif xmlFilePath == 'pennyShapedToughnessDominated' or xmlFilePath == 'pennyShapedViscosityDominated':
+         K = float(elasticParam.get('defaultBulkModulus'))
+         G = float(elasticParam.get('defaultShearModulus'))
+         youngModulus = (9.0 * K * G) / (3.0 * K + G)
+         poissonRatio = youngModulus / (2.0 * G) - 1.0
+         injectionRate = -4.0 * float(tree.find('FieldSpecifications/SourceFlux').get('scale')) / fluidDensity
+         tree = ElementTree.parse(prefix + xmlFilePath + "_benchmark.xml")
+         toughness = float(tree.find('Solvers/SurfaceGenerator').get('rockToughness'))
```

Comment:
Could you also check the python files for consistent formatting?  In a number of places the indentation isn't a multiple of four spaces

Reply:
Sure, update files as suggested